rfcs/text/0005/host.yml

```diff
@@ -0,0 +1,61 @@
+- name: host
+  fields:
+    - name: cpu.usage
+      type: scaled_float
+      scaling_factor: 1000
+      level: extended
+      short: Percent CPU used with scaling factor of 1000.
```

Comment:
The scaling factor is an ES mapping detail. I wouldn't capture that as part of the short description. I would instead recommend giving an idea of how to interpret the number. For example:

> Percent CPU used, between 0 and 1.